src/components/widgets/sales-leads-funnel/sales-leads-funnel.tmpl.html

```diff
@@ -4,7 +4,7 @@
     <h4>Widget settings</h4>
 
     <div setting-organizations parent-widget="widget" class="part" deferred="::orgDeferred" />
-    <div setting-params-picker options="statusOptions" param="status_selection" parent-widget="widget" class="part" deferred="::paramsPickerDeferred" />
+    <div setting-params-picker options="statusOptions" param="leads_per_status" has-reach="hasReach" parent-widget="widget" class="part" deferred="::paramsPickerDeferred" />
```

Comment:
I receive the parameter with the name from backend, so, if I understand correctly, I have to send it back with the same name, right?